src/components/activities/content/editors/SyllableEditor.tsx

```diff
@@ -0,0 +1,67 @@
+import ContentEditable from 'react-contenteditable';
+import React from 'react';
+
+import type { AnthemData } from 'src/activity-types/anthem.types';
+import BacklineIcon from 'src/svg/anthem/backline.svg';
+import TrashIcon from 'src/svg/anthem/trash.svg';
+import type { Activity } from 'types/activity.type';
+
+type SyllableEditorProps = {
+  update: (activity: Partial<Activity>) => void;
+  index: number;
+  backline?: boolean;
+  editable?: boolean;
+  data: AnthemData;
+};
+
+export const SyllableEditor: React.FC<SyllableEditorProps> = ({
```

Comment:
**nit:** We should avoid using the `React.FC` type. It can leads to confusion if your component does not accept children:

With `SyllableEditor: React.FC<SyllableEditorProps>`, this works even if children are not allowed.
```tsx
<SyllableEditor>
    <div>Hello!</div>
</SyllableEditor>
```

See: https://fettblog.eu/typescript-react-why-i-dont-use-react-fc/

Best is to not type components at all, so avoid `React.FC`, and use `React.PropsWithChildren` if you need children. (I know misused this already in the project, but I plan to remove all React.FC soon.)